frontend/src/views/Settings.vue

```diff
@@ -0,0 +1,141 @@
+                  <v-tooltip top>
+                    <template v-slot:activator="{ on }">
+                      <v-btn small v-on="on">
+                        <v-icon small color="primary">mdi-white-balance-sunny</v-icon>
+                      </v-btn>
+                    </template>
+                    <span>Light Mode</span>
+                  </v-tooltip>
+                  <v-tooltip top>
+                    <template v-slot:activator="{ on }">
+                      <v-btn small v-on="on">
+                        <v-icon small color="primary">mdi-weather-night</v-icon>
+                      </v-btn>
+                    </template>
+                    <span>Dark Mode</span>
+                  </v-tooltip>
+                  <v-tooltip top>
+                    <template v-slot:activator="{ on }">
+                      <v-btn small v-on="on">
+                        <v-icon small color="primary">mdi-brightness-auto</v-icon>
+                      </v-btn>
+                    </template>
+                    <span>Automatically choose theme based on your system settings</span>
+                  </v-tooltip>
+                </v-btn-toggle>
+              </v-list-item-content>
+            </v-list-item>
+            <v-list-item>
+              <v-list-item-content>
+                <v-list-item-title class="text--secondary">Log Level</v-list-item-title>
+                <v-btn-toggle v-model="logLevelIndex" dense class="pt-1">
+                  <v-btn v-for="{ value, text, icon, color } in logLevels"
+                    :key="value"
+                    small
+                    class="text-lowercase text-caption"
+                  >
+                    <v-icon small class="mr-1" :color="color">{{icon}}</v-icon>
+                    {{ text }}
+                  </v-btn>
+                </v-btn-toggle>
+              </v-list-item-content>
+            </v-list-item>
+            <v-list-item>
+              <v-list-item-content>
+                <v-switch
+                  v-model="developerMode"
+                  label="Enable developer mode"
+                ></v-switch>
+              </v-list-item-content>
+            </v-list-item>
+          </v-list>
+        </v-card>
+      </v-col>
+      <v-col cols="12" md="6">
+      </v-col>
+    </v-row>
+  </v-container>
+</template>
+
+<script>
+import { mapGetters, mapActions } from 'vuex'
+
+export default {
+  name: 'settings',
+  data () {
+    return {
+      logLevels: [
+        { value: 'debug', text: 'verbose', icon: 'mdi-bug', color: 'grey darken-4' },
+        { value: 'info', text: 'info', icon: 'mdi-information', color: 'blue darken-2' },
+        { value: 'warn', text: 'warning', icon: 'mdi-alert', color: 'warning' },
+        { value: 'error', text: 'error', icon: 'mdi-close-circle', color: 'error' },
+        { value: 'silent', text: 'silent', icon: 'mdi-pause-octagon', color: 'grey' }
+      ],
```

Comment:
as discussed, make `verbose` log level the default

Reply:
✔️